### ISSUE TITLE
Google Scholar Optimization

### ISSUE THREAD

**Issue by richardGreen** [projecthydra-labs/hydra-connect#70]
UAlberta has been struggling with getting our content indexed in Google Scholar since we moved to Sufia. However, inclusion in Scholar is absolutely critical to recruitment of content for our repository. We would love to have a discussion with others about how to improve Google Scholar indexing for Hydra repos - we can share strategies we've adopted, and would love to hear more about how others are solving this issue at their institution. 

Suggested by: Sonya Betz; UAlberta to facilitate?
Panel/Breakout
All

**Comment by richardGreen** [comment]
Which?

**Comment by richardGreen** [comment]
Yes - breakout session where we problem-solve this issue with others would be fantastic.

**Comment by richardGreen** [comment]
PANEL NEEDED

**Comment by hannahfrost** [comment]
@richardGreen I'm interested in helping with this session

**Comment by richardGreen** [comment]
Just to confirm that we are definitely still interested in hosting this breakout session on Google Scholar Indexing. Facilitators are: Sonya Betz, Weiwei Shi, and Peter Binkley, all from University of Alberta. We are fine with having the session recorded and linked to the Hydra Wiki. I've updated the session description below:

Inclusion in Google Scholar is critical to recruitment of content for our institutional repositories - from our researchers' perspective, if Google Scholar can't find it, it doesn't exist! However, when the University of Alberta migrated its institutional repository to Sufia in 2015, we saw our Google Scholar indexing plummet from nearly 6000 titles to only 35. We've since made several changes to improve our results, and are finally seeing some success as Scholar slowly returns to crawl our content. We would love to have a discussion with other institutions about improving Google Scholar indexing for Hydra repositories. Come prepared to discuss any successes and challenges you've encountered, and we will share the strategies we've adopted to improve Scholar coverage of our Sufia repository.

Do let me know if there's any more info I can provide.
